### DISCUSSION TITLE
Search with MGRS

### DISCUSSION THREAD

**Discussion by Kobrairaq** [maplibre/maplibre-gl-geocoder#131] [Q&A]
Can i Search with MGRS coordinates in reverse geocode ?

and if it's posable how to implement it in react

**Comment by HarelM** [comment]
The following is the coordinates regular expression:
https://github.com/maplibre/maplibre-gl-geocoder/blob/c893f396e3579db3bbbdebf990c659a85b896bee/lib/index.ts#L14
If the MGRS falls under this regexp, you can set the reverseGeocode method to recieve and use it.

Regarding react, this is a pure javascript library, you can eithr wrap it with react or roll your own.
